docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.5'
+
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+
+    volumes:
+      - .:/app
```

Comment:
Why mounting to app and previouly do heavy copy with "COPY Classes /app/Classes" in Dockerfile? 
I would eighter mount the stuff or copy them, if you just mount them, or in anyway, you should provide a .dockerignore with contents not to copy to the image while building process